shop/src/main/java/ro/msg/learning/shop/dto/ProductCategoryDto.java

```diff
@@ -0,0 +1,16 @@
+package ro.msg.learning.shop.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ProductCategoryDto {
+    private UUID id;
```

Comment:
do we need to give the id?

Reply:
I considered I need to be able to see the Id in the endpoint.

Reply:
ok